buffer/src/main/java/io/netty/buffer/api/AllocatorControl.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.buffer.api;
+
+/**
+ * Methods for accessing and controlling the internals of an allocator.
+ * This interface is intended to be used by implementors of the {@link BufferAllocator}, {@link Buffer} and
+ * {@link MemoryManager} interfaces.
+ */
+public interface AllocatorControl {
+    /**
+     * Allocate a buffer that is not tethered to any particular {@link Buffer} object,
+     * and return the recoverable memory object from it.
```

Comment:
```suggestion
     * Allocates a buffer that is not tethered to any particular {@link Buffer} object,
     * and returns the recoverable memory object from it.
```

Think like this - you omit `This method` at the beginning of the summary sentence.